ndr_fasttransfer.c

```diff
@@ -288,10 +326,10 @@ static int ndr_parse_propValue(TALLOC_CTX *mem_ctx, struct ndr_print *ndr,
 
```

Comment:
retval checks are initially missing here. They now have an impact given that propValue isnever checked before it is used in a further talloc_ call within ndr_parse_namedproperty